pkg/operator2/starter.go

```diff
@@ -122,6 +122,14 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		return err
 	}
 
+	// add syncing for the console-config ConfigMap (indirect watch for changes)
```

Comment:
Why is this not just another config map with `config.openshift.io/inject-trusted-cabundle` in `manifests`?

Reply:
I'll make it so